scripts/update-changelog

```diff
@@ -46,4 +46,7 @@ else
     fi
 fi
 
+# Normalize version, convert any - to ~, e.g. 0.9.0-rc1 to 0.9.0~rc1
+PKG_VERSION=$(echo $PKG_VERSION | sed 's/-/~/')
```

Comment:
Nit: Without the `g` flag, this is actually:

```suggestion
# Normalize version, convert the first - to ~, e.g. 0.9.0-rc1 to 0.9.0~rc1
PKG_VERSION=$(echo $PKG_VERSION | sed 's/-/~/')
```

E.g., `PKG_VERSION=0.10.0-rc1 ./scripts/update-changelog securedrop-client` yields `0.10.0~rc3-47+bullseye` rather than `0.10.0~rc3~47+bullseye`.

Which do you intend here, @legoktm?  (You'd know better than I whether multiple `~` separators are even valid in a version string. :-)

Reply:
Oh good point, we want `/g` here, no dashes anywhere but multiple tildes are fine.

We should also be able to get dpkg to error out if a non-native package is used, let me see...